tpu-client-next/Cargo.toml

```diff
@@ -0,0 +1,34 @@
+[package]
+name = "solana-tpu-client-next"
+description = "Client code to send transaction to TPU."
+version = { workspace = true }
+authors = { workspace = true }
+repository = { workspace = true }
+homepage = { workspace = true }
+license = { workspace = true }
+edition = { workspace = true }
+
+[dependencies]
+async-trait = { workspace = true }
+log = { workspace = true }
+lru = { workspace = true }
+quinn = { workspace = true }
+rustls = { workspace = true }
+solana-connection-cache = { workspace = true }
```

Comment:
lol, it sucks to have to depend on this for 

```
enum Protocol {
	TCP,
	UDP
}
```

we should do something about it

Reply:
In general, I think we should trim down all these deps as much as possible.
Break the cycle of violence!

Reply:
it is to have leader estimation service. A separate task on my roadmap to cover that component with tests, move it out from solana-connection-cache and give it some love in general.